pypeit/spectrographs/keck_kcwi.py

```diff
@@ -1282,14 +1281,13 @@ def get_rawimage(self, raw_file, det):
             (1-indexed) number of the amplifier used to read each detector
             pixel. Pixels unassociated with any amplifier are set to 0.
         """
-        # Check for file; allow for extra .gz, etc. suffix
-        fil = glob.glob(raw_file + '*')
-        if len(fil) != 1:
-            msgs.error("Found {:d} files matching {:s}".format(len(fil), raw_file))
+        fil = utils.find_single_file(f'{raw_file}*')
+        if fil is None:
```

Comment:
Should this if statement be located in `utils.find_single_file`?

Reply:
Good idea.  There are pieces of the code that use the fact that the function returns None for files that don't exist.  So I added a keyword to indicate that the code should fault if it can't find a relevant file.